### ISSUE TITLE
Enable native spellcheck

### ISSUE THREAD

**Issue by atd** [P2Pvalue/swellrt#78]
Spellcheck in editor is disabled:

https://github.com/P2Pvalue/swellrt/blame/master/src/org/waveprotocol/wave/client/editor/EditorImpl.java#L1748

May we try to turn it on? This feature was requested by one user Pear2Pear https://github.com/P2Pvalue/pear2pear/issues/109

**Comment by llwp** [comment]
Is this still an active need/request?

**Comment by atfornes** [comment]
As for Teem (former Pear2Pear) is a non prioritized feature (maybe we will not even need it).

I do not know how important is this feature for other SwellRT projects.

**Comment by atd** [comment]
@atfornes I don't understand your answer. Of course we can live without this, but the spellcheck working in Teem pads would be a nice thing to have IMO. I don't think the user that requested that feature has changed her mind

I am still wondering the rationale behind SwellRT disabling the spellchecker

**Comment by atfornes** [comment]
@atd, as far as I know, we have not prioritized this feature in our Sprint meetings neither we have detected this as an important need of our users.

To the question, is this an active need? I reply that form teem is not currently active or a need. 
However I let the open question on how is this important to other SwellRT projects. Personally I believe it is important to be activated in that broader context, and even that it will be good for Teem.

 Please let me know if you think otherwise.

**Comment by atd** [comment]
@atfornes we did receive comments from our users missing the spellchecker. Still wondering why we have to remove it

**Comment by atfornes** [comment]
@atd, I was not aware of this comments despite the original one. I did not say we have to remove it, just that is not currently prioritized, and that maybe we will not need it (in the sense of user needs of Lean approach we are following).

With the information you are sharing, it is clear that the priority of this issue is not as low a I commented, thanks for bringing it here.

The resolution of this issue will benefit Teem and other SwellRT project. I believe that it is an important feature of SwellRT.
